demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -1324,6 +1360,12 @@ int main( int argc,
      * done only once in this demo. */
     returnStatus = initializeMqtt( &mqttContext, &networkContext );
 
+    /* Seed pseudo random number generator with current time.
+     * Note: The random number generator is used for adding randomness
+     * to the client ID strings to avoid client identifier collision
+     * when connecting to the MQTT broker. */
+    srand( time( NULL ) );
```

Comment:
I think in CI it is very likely that we start 2 instances within 1 second, which will likely cause a lot of collisions.